apps/builder/app/builder/features/sidebar-left/navigator/css-preview.tsx

```diff
@@ -73,11 +73,11 @@ export const CssPreview = () => {
     <CollapsibleSection label="CSS Preview" fullWidth>
       <ScrollArea css={{ px: theme.spacing[9] }}>
         <pre tabIndex={0} className={preStyle()}>
-          <code
```

Comment:
that's super weird, why?

Reply:
have zero ideas, same you get in react if p tag inside p, have no idea whats wrong here